frontendservice/package.json

```diff
@@ -12,6 +12,7 @@
     "bulma": "^0.9.0",
     "core-js": "^3.6.5",
     "firebase": "^7.15.0",
+    "js-cookie": "^2.2.1",
```

Comment:
Why is this needed?

Reply:
I had just pulled the entire code from master and this change came automatically. I haven't written this or used it.

Reply:
remove this then?

Reply:
Done.